scripts/gen_jct_vc.py

```diff
@@ -39,13 +39,16 @@
 BITSTREAM_EXTS = ('.bin', '.bit', '.264', '.h264',
                   '.jvc', '.jsv', '.jvt', '.avc', '.26l')
 MD5_EXTS = ('yuv.md5', '.md5', 'md5.txt')
-MD5_EXCLUDES = ('__MACOSX', '.bin.md5', 'bit.md5')
-RAW_EXTS = ('.yuv', '.qcif')
+MD5_EXCLUDES = ('bit.md5')
```

Comment:
There are files with the bin.md5 extension, those are bitstream input files and they shouldn't be used to find the decoded results md5.

Reply:
My bad. I read `.bit.md5` and `bit.md5` and thought it was redundant